skimage/filters/ridges.py

```diff
@@ -105,20 +112,18 @@ def compute_hessian_eigenvalues(image, sigma, sorting='none'):
         of the input image.
     """
 
-    # Import has to be here due to circular import error
-    from ..feature import hessian_matrix, hessian_matrix_eigvals
-
     # Convert image to float
     image = img_as_float(image)
 
     # Make nD hessian
-    hessian_elements = hessian_matrix(image, sigma=sigma, order='rc')
+    hessian_elements = hessian_matrix(image, sigma=sigma, order='rc',
+                                      mode=mode)
```

Comment:
Shouldn't you pass cval in here too, in case `mode='constant'`?

Reply:
Fair enough! ;-) Sorry for the delay...